### ISSUE TITLE
fd segmentation faults

### ISSUE THREAD

**Issue by gwijayas** [termux/termux-packages#2921]
Getting segmentation faults on executing `fd`. My device:

```
$ termux-info
Updatable packages:
All packages up to date
System information:
Linux localhost 3.10.20-x86_64_moor-265462-gfd33dd4 #1 SMP PREEMPT Fri Sep 30 15:31:23 CST 2016 x86_64 Android
Termux-packages arch:
i686
Android version:
5.0
Device manufacturer:
asus
Device model:
ASUS_Z00AD
```

**Comment by fornwall** [comment]
@its-pointless Was it an issue with android versions earlier than 7.0 (where the platform did not use jemalloc)? Was that for all the arches?

**Comment by xeffyr** [comment]
@fornwall It works on ARM with Android 6.0.

**Comment by Grimler91** [comment]
I tried it in android 7.1 i686 emulator, gives a segfault there as well.
I built a debug variant of fd but gdb gives no output ("process exited normally") and valgrind only complains about stuff in /system/bin/linker so I'm not sure what to make of it

**Comment by fornwall** [comment]
Tried in android 9.0 i686 emulator, worked there.

**Comment by Herzogs** [comment]
I do not know if my information is useful but in version 5.0 (arm) it works

**Comment by its-pointless** [comment]
I686 
needed to specify it to use jemalloc ripgrep. 
Compiling on device worked regardless try this 
https://github.com/alexcrichton/jemallocator

**Comment by its-pointless** [comment]
#2930

**Comment by fornwall** [comment]
Should be fixed in version `7.2.0-1` of the fd package, update with `pkg up`.

This version of the package was rebuilt with version 1.32 of the rust toolchain.
